### DISCUSSION TITLE
Make an option to show what is person generating

### DISCUSSION THREAD

**Discussion by Kotoezhh** [davehornik/sd-discord-rich_presence#10] [Ideas]
As image generates it will pull results into discord 
![image](https://github.com/davehornik/sd-discord-rich_presence/assets/98416198/af58567e-4b4c-4824-9437-088ddcbd20ed)
Basically this but in discord 
idk why but it would be cool to see :p

**Comment by davehornik** [comment]
This is both unsafe (in case of doing NSFW), but also privacy-invading.
Even as an option, extension would have to upload those images (-consider someone with very low connectivity), then store them somewhere safe to prevent leaks, and then run clean-process after it's no longer being generated.

So, this whole idea, is very bad and unpolished.

**Reply by Kotoezhh** [reply]
i mean i imagined it as optional for a reason 
if you want and can use it why not
if you dont - just dont
but you do you, i just gave an idea 
gl with extension anyway
